src/dijkstra.rs

```diff
@@ -97,6 +97,7 @@ pub fn dijkstra<G, F, K>(
     start: G::NodeId,
     goal: Option<G::NodeId>,
     mut edge_cost: F,
+    mut path: Option<&mut HashMap<G::NodeId, Vec<G::NodeId>>>,
```

Comment:
Don't need to update docs?

Reply:
Yeah, I probably should, even though it's internal for retworkx we should still keep it documented

Reply:
Added in: https://github.com/Qiskit/retworkx/pull/162/commits/c16f64b2d1b5c90bb7b9ebb5f729fdf1e503e740